src/index.html

```diff
@@ -274,15 +274,18 @@ <h1>Easy Steps To Finding Your Perfect Roommate</h1>
                     border-style: solid; 
                     border-width: 1px;
                     border-color: white;
+                }
+                    
                 }
 			</style>
 			<div class="container animated bounce">
 				<h1>Congratulations!!</h1>
 				<p>We have received your invitation request to be a member of slash the rent.</p>
 				<p>Please be patient while we process your request. You will receive an invitation response via email/text.</p>
-				<app-logo color="white"></app-logo>
+				<app-logo type="white"></app-logo>
```

Comment:
why is  app-logo having an attribute of type? Change it back to color

Reply:
will do, that was through testing and experimenting on the logo.mjs

Reply:
completed